### PR TITLE
Fronden design team has completed work on Milestone 2nd

### PR DESCRIPTION
**Pull request by kshekh**
The frontend team has completed the frontend task on 2nd milestone.
The page Customer List "/customers-list"
The page Customer List "/customers-edit"
These are only for looking out fronted view you can remove them when they will be merged original customer page

### CODE REVIEW

**Comment by paulchrisluke** [comment]
Missing notes components
Screenshot 2022-12-25 at 2 47 20 PM

Minor spelling issues on customer-list @kshekh should be "search for customer" and "sort"
Screenshot 2022-12-25 at 2 48 52 PM

File: src/components/AddCustomer.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                <div class="overflow-hidden shadow-3 rounded-lg">
+                  <div class="space-y-6 bg-white px-4 py-5 sm:p-10">
+                    <Textarea name="text-note" rows="4" id="textNote" label-text="Note" />
+                  </div>
+                </div>
+              </form>
+            </div>
+          </div>
+          <div class="pt-7 flex items-center mt-3">
+            <Button @emitclick="consoleClick" class="sm:min-w-40 " :btn-primary="true">Add customer</Button>
```
Comment (giteshk11) [inline src/components/AddCustomer.vue:53]: `emitclick` should be `click`

File: src/components/CreateAccount.vue [hunk 2]
```diff
@@ -156,7 +158,7 @@
             >
               <div class="flex items-center w-full xs:w-auto">
                 <!-- This is for emit method -->
-                <Button type="submit" @emitclick="consoleClick">
+                <Button type="submit" @emitclick="consoleClick" :btn-primary="true">
```
Comment (giteshk11) [inline src/components/CreateAccount.vue:161]: `emitclick` should be `click`

File: src/components/ForgotPassword.vue [hunk 3]
```diff
@@ -43,7 +43,7 @@
 
               <div class="flex items-center justify-between">
                 <div class="flex items-center">
-                  <Button type="submit" :on-click="consoleClick">
+                  <Button type="submit"  @emitclick="consoleClick">
```
Comment (giteshk11) [inline src/components/ForgotPassword.vue:46]: `emitclick` should be `click`

File: src/components/HomeCustomer.vue [hunk 4]
```diff
@@ -75,12 +75,8 @@
                       class="space-y-5"
                     >
                       <p class="text-md text-grey-800">{{ step.answer }}</p>
-                      <button
-                        type="submit"
-                        class="mt-4 flex w-auto justify-center rounded-md border border-transparent bg-peach py-2 px-4 text-sm font-semibold text-grey-800 shadow-sm hover:bg-peach-2 focus:outline-none focus:ring-2 focus:ring-peach focus:ring-offset-2 transition-all ease-in-out duration-300"
-                      >
-                        Add customers
-                      </button>
+                      <Button @emitclick="consoleClick" class="sm:min-w-40 mt-4" :btn-primary="true">Add customer</Button>
```
Comment (giteshk11) [inline src/components/HomeCustomer.vue:78]: `emitclick` should be `click`

**Review by giteshk11 — CHANGES_REQUESTED** [review]

File: src/components/AddCustomer.vue (hunk 1 above)
Comment (kshekh) [reply src/components/AddCustomer.vue:53]: @giteshk11 I got the issues. I will fix it and let you know.

File: src/components/CreateAccount.vue (hunk 2 above)
Comment (kshekh) [reply src/components/CreateAccount.vue:161]: @giteshk11 I got the issues. I will fix it and let you know.

File: src/components/ForgotPassword.vue (hunk 3 above)
Comment (kshekh) [reply src/components/ForgotPassword.vue:46]: @giteshk11 I got the issues. I will fix it and let you know.

File: src/components/HomeCustomer.vue (hunk 4 above)
Comment (kshekh) [reply src/components/HomeCustomer.vue:78]: @giteshk11 I got the issues. I will fix it and let you know.

**Review by giteshk11 — APPROVED** [review]

**Pull request merged**